app/views/docs/authentication.phtml

```diff
@@ -307,6 +307,11 @@ try await account.createOAuth2Session(provider: "amazon")</code></pre>
     If there is already an active anonymous session, the new session will be attached to it. If there are no active sessions, the server will attempt to look for an account with the same email address as the email received from the OAuth2 provider and attach the new session to the existing account. If no matching account is found - the server will create a new account.
 </p>
 
+<div class="notice">
+    <h3>Note</h3>
```

Comment:
On second thought, can you suggest some better titles for this than "Note"? We don't use generic titles like this.